### ISSUE TITLE
Does ogre-meshviewer have a way to set the rendersystem and its options

### ISSUE THREAD

**Issue by sercero** [OGRECave/ogre-meshviewer#17]
Hello,

I wasnt able to set the Rendersystem and its options from ogre-meshviewer.

I have to manually edit the config file whose location I don't know.

I'm not sure if the default ogre config dialog is shown at some point, but it is not showing for me.

**Comment by paroj** [comment]
The config dialog is disabled in ogre-python on linux to avoid pulling in Xaw. This was not an issue so far, as the snap package comes with Xaw enabled.

However, it would be more sane to port the config dialog to ImGui for multiple reasons. I had that on my mind for a while, but took this chance to create an issue for it now: https://github.com/OGRECave/ogre/issues/3025

Anyway, the config file is at `~/.cache/OgreMeshViewer/`

**Comment by sercero** [comment]
OK, I'm trying to implement a simple dialog on ImGui to create this.

But I'm stumped on a ver simple thing: 
```
ImGui.Combo("combo", item_current, ["1", "2", "3"], 3)

TypeError: Wrong number or type of arguments for overloaded function 'Combo'.
  Possible C/C++ prototypes are:
    ImGui::Combo(char const *,int *,char const *const [],int,int)
    ImGui::Combo(char const *,int *,char const *const [],int)
    ImGui::Combo(char const *,int *,char const *,int)
    ImGui::Combo(char const *,int *,char const *)
    ImGui::Combo(char const *,int *,bool (*)(void *,int,char const **),void *,int,int)
    ImGui::Combo(char const *,int *,bool (*)(void *,int,char const **),void *,int)
```

I don't understand how to pass arguments to ImGui::Combo and there aren't any examples of that.

**Comment by paroj** [comment]
yeah.. this one is not properly wrapped. `List[str]` in python does not convert to `const char* items[]` - it needs a special typemap.
Will fix this in ogre, however this will become available only after the next release.

BeginCombo()/EndCombo() might work for you meanwhile. https://github.com/ocornut/imgui/issues/1658

**Comment by paroj** [comment]
actually, wrapping `const char* items[]` requires some lengthy python specific code:
https://www.swig.org/Doc3.0/Python.html#Python_nn59

while BeginCombo()/EndCombo() already works and seems cleaner too. So I would leave it at that.

**Comment by sercero** [comment]
Yes, I'm using `BeginCombo()/EndCombo()` and it works.

Thanks!
